exercises/Sql-1-R.md

```diff
@@ -11,13 +11,14 @@ It is clear Dr. Undómiel appreciates your skill working with large databases an
 she seems to expect you will maintain your benevolence. (*Such is a fair 
 expectation of a true wizard*). This time though, she's looking for some extra 
 detail in her queries. She's curious if desert rodents are [dimorphic](https://en.wikipedia.org/wiki/Sexual_dimorphism) in size.
- 
 
 1. Download a new copy of the [Portal database](http://files.figshare.com/2292171/portal_mammals.sqlite). 
-2. Connect to `portal_mammals.sqlite` using the `RSQLite` package. 
-3. Select and print out the average hind foot length and average weight of:
-  - all *Dipodomys spectabilis* individuals
-  - male *D. spectabilis*
-  - female *D. spectabilis*.
-
-
+2. Connect to `portal_mammals.sqlite` using the `RSQLite` package.
+3. Start by reminding yourself of what tables are in the database using
+`dbListTables`
```

Comment:
I made a personal style decision to include `()` when naming functions in the exercises. In this case, '`dbListTables()`'.